OpenEXR/IlmImfTest/testHuf.cpp

```diff
@@ -186,6 +221,7 @@ testHuf (const std::string&)
 	Array <unsigned short> raw (N);
 
 	fill1 (raw, N, 1, rand48);	  // test various symbol distributions
+        compressVerify(raw, N, 3762495704U);
```

Comment:
Could you add a comment here explaining how someone might produce this number? It's worrying to have opaque magic sitting there because we don't know how to write new tests, or modify them. It would be good not to use rand48 here as well, rather something that is cross platform reproducible.

Reply:
Thanks for having a look... agree with the observation. I was attempting to piggy back that "fill1" test case... ;). but let me redo it and update this PR... by making a new "fill6" which is sequentially unique set of numbers (that exposes the "fix" to sorting equally entries in the decode table since each number/code has a freq of one). This makes the fill6 raw data a known series of numbers (rather than a rand48 one) and so that its huf compressed hash  (DEK hash) pre-computed aprior and more descriptively stated in code i.e. its the DEA hash for huf compress a series of N numbers from  1...N.

Reply:
Updated the PR. Turns out fill4() (and fill5()) were suitable non-random data sets; so I applied the compressVerify() check on those tests. 
Also added a comment on how to determine the DEK hashes apriori.

Reply:
Thanks very much! The mystery is lifted :)

Reply:
Mystery resolved! Thanks very much :)